source/ImageBuffer.cpp

```diff
@@ -223,7 +223,8 @@ namespace {
 		// If the buffer is not yet allocated, allocate it.
 		try {
 			buffer.Allocate(width, height);
-		} catch (const bad_alloc &) {
+		} catch (const bad_alloc &)
+		{
```

Comment:
I would think this catch and the one below it would be formatted as either
```
} catch () {
```
or
```
}
catch ()
{
```
not the in-between of
```
} catch ()
{
```

There's also the case of
```
try {
```
above this that could perhaps be
```
try
{
```

Reply:
In the style guide, MZ said `{` should go on a new line only when it is following a `)`, so, same line when opening a namespace, or class, or a try, but new line after the `catch()`.
I'm not sure what it said about if the about if the `catch` should be on a new line after the `}`.

Reply:
The actual cases weren't always the same, but I agree with putting catch on a new line. Added a new commit to fix this in all files (except .hpp files, that is).